app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,61 @@
+class PasswordResetsController < ApplicationController
+   before_action :find_user, :valid_user, :check_expiration,  only: [:edit, :update]
+
+  def new
+  end
+
+  def create
+    @user = User.find_by email: params[:password_reset][:email].downcase
```

Comment:
em giải thích cho anh action này nhé, toàn bộ action create, ko phải chỉ mỗi dòng này